phy/traces/detect.py

```diff
@@ -99,7 +99,7 @@ def detect(self, data_t, threshold=None):
         if isinstance(threshold, string_types):
             assert threshold in self._thresholds
             threshold = self._thresholds[threshold]
-        threshold = float(threshold)
+        # threshold = float(threshold)
```

Comment:
* bug when this is an array
* not sure why I needed this!
* should actually be removed!